Frontend/ui-driver/src/Screens/ApplicationStatusScreen/View.purs

```diff
@@ -137,17 +129,79 @@ applicationStatus state push =
   [ orientation VERTICAL
   , width MATCH_PARENT
   , padding (Padding 16 0 16 24)
-  , margin (MarginTop 50)
-  , height WRAP_CONTENT
-  ][  textView
-      [ text (getString APPLICATION_STATUS)
-      , textSize FontSize.a_14
-      , visibility if state.data.dlVerificationStatus == "PENDING" && state.data.rcVerificationStatus == "PENDING" then GONE else VISIBLE
-      , color Color.black800
-      , fontStyle $ FontStyle.regular LanguageStyle
-      ]
+  , margin (MarginTop 30)
+  , height MATCH_PARENT
+  ][  linearLayout
+            [ width MATCH_PARENT
+            , height MATCH_PARENT
+            , margin $ MarginBottom 20
+            ][
+               textView
+              [   width WRAP_CONTENT
+               , height WRAP_CONTENT
+               , text ("Start earning in 3 simple step")--"Start earning in 3 simple step"
+               , textSize FontSize.a_14
+               , fontStyle $ FontStyle.medium LanguageStyle
+              ]
+              ,  textView
+              [  width MATCH_PARENT
+               , height WRAP_CONTENT
+               , gravity RIGHT
+               , text (show (completePercentage state) <> " % " <> ("Complete"))
```

Comment:
try to get the text via getString method and define its translations accordingly in Strings folder.